researcher_ids/RI_Claims_V1.md

```diff
@@ -202,58 +205,62 @@ Additonal terms on fields:
-        /userinfo endpoint based on the "scope" claim as outlined elsewhere
-        within this specification.
-
-    3.  The Identity Broker signs the access token, making it a Passport.
-
-    4.  The Identity Broker introduces new RI Claim Objects as a
-        [Root Claim Broker](https://github.com/ga4gh/data-security/blob/master/AAI/AAIConnectProfile.md#term-root-claim-broker)
-        by encoding objects within the "ga4gh" OIDC claim (available via
-        /userinfo) whereas the Identity Broker propagates RI Claims that it
-        received from upstream brokers using
-        [Embedded Passport Tokens](#embedded-passport-tokens).
-
-    5.  The Passport for the /userinfo request MUST contain "ga4gh" as a space-
-        separated sub-string within the "scope" claim in
-        order to include the "ga4gh" OIDC claim as part of the response.
-
-    6.  The Identity Broker MUST NOT include claims from upstream Identity
-        Brokers as part of the "ga4gh" OIDC claim unless the Identity Broker
-        wishes to follow an implicit trust security model and take
-        responsibility for presenting those upstream claims to a specific
-        audience. For example, an organization may provide an
-        intraorganizational security service that filters and flattens all
-        claims that meet that organization’s security and trust model on behalf
-        of many Claims Clearinghouses within that organization where each such
-        client of this service places absolute trust in its ability to filter
-        claims into a flattened claim set that gets signed as a Passport.
+        and MUST NOT include the "ga4gh" claims within the Passport Access
+        Token. The RI Claims are only available at the /userinfo endpoint
+        based on the "scope" claim as outlined elsewhere within this
+        specification.
+
+    3.  The Identity Broker introduces new RI Claim Embedded Tokens from a
+        [Claim Signatory](https://github.com/ga4gh/data-security/blob/master/AAI/AAIConnectProfile.md#term-embedded-claim-signatory)
+        by encoding RI Claim Objects within the "ga4gh" OIDC claim (available
+        via /userinfo). If the Broker is the Claim Signatory, it MUST set
+        the "iss" to itself and sign such Embedded Tokens with its own
+        private key as described in the
+        [AAI Specification](https://github.com/ga4gh/data-security/blob/master/AAI/AAIConnectProfile.md).
+
+    4.  The Identity Broker propagates RI Claims that it received
+        from upstream Brokers by copying each RI Claim Object Embedded
+        Token string into the coorsponding RI Claim within the outer "ga4gh"
+        OIDC claim. These strings are provided unmodified, however they
+        MAY be filtered as described elsewhere in this specification.
+
+    5.  The Identity Broker signs the JWT, making it a Passport Access
+        Token.
+
+    6.  The Passport for the /userinfo request MUST contain "ga4gh" as a
```

Comment:
The "Passport" as defined above is not an access token, thus "the passport for the /userinfo request" is a non-defined thing.
It should probably be "access token", but in fact there is no need to have the "scope" claim inside of the access token, it is not used by the Authorization Server itself.

Reply:
Rephrased